.github/workflows/release-temp.yml

```diff
@@ -0,0 +1,35 @@
+name: Temp Publish Node.js (shoryukyzan/hot-module-reload)
+
+on:
+  push:
+    branches:
+      - "shoryukyzan/hot-module-reload"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+          registry-url: https://npm.pkg.github.com/
+          scope: "@shoryukyzan"
```

Comment:
TIL!

If you're just publishing to test, check out [git-publish](https://github.com/privatenumber/git-publish). It works like `npm publish`, but pushes to a GitHub branch.